### ISSUE TITLE
panic: runtime error: index out of range during push

### ISSUE THREAD

**Issue by cgogolin** [odeke-em/drive#618]
I just tried to push a large number of files to my Drive with the version available via 

`go get -u github.com/odeke-em/drive/drive-google`

and get a `panic: runtime error: index out of range` error.
```
[...]
+ ./library/some.pdf
Addition count 2135 src: 2.58GB
Proceed with the changes? [Y/n]:Y
0 / 2772573420 [---------------------------------------------------------------------------------------------------------------------------------------------]   0.00%
panic: runtime error: index out of range
[...]
```

I am attaching the [full output](https://github.com/odeke-em/drive/files/210005/drivelog.txt).

Pulling my current Drive worked flawlessly.

Maybe it is relevant that ./library is a symbolic link to a directory on the same partition but outside of the "working copy" of drive?

**Comment by odeke-em** [comment]
Hello there @cgogolin, thank you for reporting this and welcome to drive.

If you don't mind, an actual `drive version` would be helpful and perhaps a `git log | head -n 3` insde $GOPATH/src/github.com/odeke-em/drive would do. Thanks for the stack trace!

**Comment by cgogolin** [comment]
Here you are:
```
~/drive$ drive-google version
drive version: 0.3.5
Commit Hash: <CURRENT_COMMIT>
Go Version: <GO_VERSION>
OS: <OS_INFO>
BuildTime: 
```
```
~/.config/go/src/github.com/odeke-em/drive$ git log | head -n 3
commit 7c654c944f21a58ce090d6d752e4a42bc576c809
Author: Emmanuel Odeke <emm.odeke@gmail.com>
Date:   Thu Apr 7 00:38:24 2016 -0700
```

**Comment by odeke-em** [comment]
Awesome! Thank you @cgogolin. I couldn't catch sleep so I've dug into it and discovered the bug came from PR https://github.com/odeke-em/drive/pull/616. I was led there due to the output of your manifest ie
`lib/*` instead of the usual `/lib/*` which makes sense for those panics because path splitting is dependant upon the root '/' first existing and then searching on from then on.
PR coming up in a few.

**Comment by odeke-em** [comment]
Hello again @cgogolin, this issue was auto-closed by PR #619. Please get the latest from master as I'll be going to try to sleep for a few, before a long day during which I won't have time during the day to merge the PR. Thank you for the report and for the patience. Let me know how it goes.

**Comment by cgogolin** [comment]
Wow! I wish developers would always respond like that!
I am now at 
```
~/.config/go/src/github.com/odeke-em/drive$ git log | head -n 3
commit 24c1ff64ec2d48ca75fb1a5341c7b7076a315048
Author: Emmanuel Odeke <emm.odeke@gmail.com>
Date:   Fri Apr 8 06:15:50 2016 -0700
```
and pull seems to work smoothly.
